site/utils/formUtils.ts

```diff
@@ -72,3 +73,5 @@ export const getDataInPages = <T>(pageNumber: number, data: T[]): T[] => {
     const endPoint = pageNumber * 10;
     return data.slice(startPoint, endPoint);
 };
+
+export const toLowerStartCase = (text: string) => startCase(text.toLowerCase());
```

Comment:
I'd maybe put this in the main utils file since we already have some other string formatting functions in there

Reply:
done